apps/bytebank-native/app/(tabs)/index.tsx

```diff
@@ -1,60 +1,205 @@
+
+import type { ITransaction } from '@fiap-tech-challenge/database/types';
+
+export default function HomeScreen() {
+  const { 
+    stats, 
+    recentTransactions, 
+    loading,
+    hasStats,
+    refreshData
+  } = useHomeDashboard();
+
+  const { handleSyncTransactions } = useTransactions();
+  const { user } = useAuth();
+
+  const [refreshing, setRefreshing] = React.useState(false);
+  const [selectedTransaction, setSelectedTransaction] = React.useState<Transaction | null>(null);
+  const [showTransactionModal, setShowTransactionModal] = React.useState(false);
+  const [showCreateModal, setShowCreateModal] = React.useState(false);
+
+  useFocusEffect(
+    React.useCallback(() => {
+      refreshData();
+    }, [refreshData])
+  );
+
+  const handleRefresh = React.useCallback(async () => {
+    setRefreshing(true);
+    try {
+      await refreshData();
+    } catch (error) {
+      console.error('Error refreshing home data:', error);
+    } finally {
+      setRefreshing(false);
+    }
+  }, [refreshData]);
+  const handleTransactionPress = (transaction: Transaction) => {
+    setSelectedTransaction(transaction);
+    setShowTransactionModal(true);
+  };
+
+  const handleViewAllTransactions = () => {
+    router.push('/transactions');
+  };
+
+  const handleCloseTransactionModal = () => {
+    setShowTransactionModal(false);
+    setSelectedTransaction(null);
+  };
+
+  const handleCreateTransaction = () => {
+    setShowCreateModal(true);
+  };
+
+  const handleTransactionCreated = (newTransaction: ITransaction) => {
+    const localTransaction: Transaction = mapITransactionToTransaction(newTransaction);
+    
+    handleSyncTransactions(localTransaction, 'create');
+    
+    refreshData();
+    
+    setShowCreateModal(false);
+  };
+
+  const handleCloseCreateModal = () => {
+    setShowCreateModal(false);
+  };
 
-export default function TransactionsScreen() {
   return (
-    <ParallaxScrollView
-      headerBackgroundColor={{ light: '#553860', dark: '#553860' }}
-      headerImage={
-        <IconSymbol
-          size={310}
-          color="#8A6AA1"
-          name="arrow.right.arrow.left"
-          className="-bottom-[60px] -left-[35px]"
+    <SafeAreaView className="flex-1 bg-white">
+      <ScrollView 
+        className="flex-1"
+        refreshControl={
+          <RefreshControl 
+            refreshing={refreshing} 
+            onRefresh={handleRefresh} 
+          />
+        }
+        showsVerticalScrollIndicator={false}
+      >
+        <View 
+          className="flex-col items-center w-full"
+          style={{ 
+            backgroundColor: Colors.light.primary + '15',
+            paddingTop: 28,
+            paddingBottom: 28,
+            paddingLeft: 40,
+            paddingRight: 40,
+          }}
+        >
+          <View className="flex-col items-start self-stretch" style={{ gap: 4 }}>
```

Comment:
Por favor, usar tailwind css pra manter o padrão

```suggestion
          <View className="flex-col items-start self-stretch gap-1">
```